### PR TITLE
Update containerd v1.5.4

### PR DESCRIPTION
**Pull request by thaJeztah**
Update to containerd v1.5.4 to address [CVE-2021-32760][1].

[1]: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-32760

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Review by samuelkarp — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
@tonistiigi @cpuguy83 ptal

File: vendor.conf [hunk 1]
```diff
@@ -128,7 +128,7 @@ github.com/googleapis/gax-go                        bd5b16380fd03dc758d11cef74ba
 google.golang.org/genproto                          3f1135a288c9a07e340ae8ba4cc6c7065a3160e8
 
 # containerd
-github.com/containerd/containerd                    0e8719f54c6dc6571fc1170da75a85e86c17636b # v1.5.3
+github.com/containerd/containerd                    69107e47a62e1d690afa2b9b1d43f8ece3ff4483 # v1.5.3
```
Comment (cpuguy83) [inline vendor.conf:131]: Comment is incorrect.

Comment (thaJeztah) [reply vendor.conf:131]: Whoops. thx; fixed!

**Review by AkihiroSuda — APPROVED** [review]

**Comment by thaJeztah** [comment]
One failure on arm64 (known flaky: https://github.com/moby/moby/issues/42582)

```
=== RUN   TestCreateParallel
time="2021-07-20T07:10:40Z" level=warning msg="bridge store not initialized. kv object docker/network/v1.0/bridge/net99/ is not added to the store"
time="2021-07-20T07:10:40Z" level=warning msg="bridge store not initialized. kv object docker/network/v1.0/bridge/net75/ is not added to the store"
    bridge_test.go:1133: Success should be 1 instead: 2
--- FAIL: TestCreateParallel (0.37s)
```

re-kicked CI

**Pull request merged**